psl-core/src/main/java/org/linqs/psl/application/inference/online/OnlineServer.java

```diff
@@ -72,6 +83,21 @@ public void start() {
         serverThread.blockUntilReady();
     }
 
+    /**
+     * Create a temporary file in the default temporary file directory.
+     * The creation of the temporary file may be used as a signal that this server is
+     * ready to accept client connections and online actions.
+     */
+    private void createServerTempFile() {
+        try {
+            tmpFile = File.createTempFile("onlinePSLServer", ".tmp");
+        } catch (IOException ex) {
+            throw new RuntimeException(ex);
+        }
+        tmpFile.deleteOnExit();
+        log.info("Temporary server config file at: " + tmpFile.getAbsolutePath());
```

Comment:
This log should probably be downgraded to debug (since most users will never care about it).